### PR TITLE
Embeddable docs content 

### PR DESCRIPTION
**Pull request by caylahamann**
embeddable pages!

### CODE REVIEW

File: gatsby-config.js [hunk 1]
```diff
@@ -405,6 +405,7 @@ module.exports = {
         allPageHeaders: [
           'Referrer-Policy: no-referrer-when-downgrade',
           'Content-Security-Policy: frame-ancestors *.newrelic.com',
+          'X-Frame-Options SAMEORIGIN',
```
Comment (caylahamann) [inline gatsby-config.js:408]:
```suggestion
          'X-Frame-Options: SAMEORIGIN',
```

**Comment by roadlittledawn** [comment]
let's make sure these pages are hidden from swiftype and google and other robot overlords

File: gatsby/on-pre-render-html.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+const onPreRenderHTML = ({
+  getHeadComponents,
+  replaceHeadComponents,
+  pathname,
+}) => {
+  const headComponents = getHeadComponents();
+  if (pathname.endsWith('/embed') || pathname.endsWith('/embed/')) {
+    return replaceHeadComponents([
+      headComponents.filter(
```
Comment (LizBaker) [inline gatsby/on-pre-render-html.js:11]: remove outer array

File: src/layouts/EmbedLayout.js [hunk 3]
```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { Layout } from '@newrelic/gatsby-theme-newrelic';
+import PropTypes from 'prop-types';
+import { css, Global } from '@emotion/react';
+import { Helmet } from 'react-helmet';
+
+const EmbedLayout = ({ children }) => {
+  return (
+    <>
+      <Helmet>
```
Comment (LizBaker) [inline src/layouts/EmbedLayout.js:10]: find out why this tag isn't showing up

**Comment by LizBaker** [comment]
had to do a cache clear for the noindex meta tag to show up: https://build-3346fe61-6ff6-4fed-925a-03c65c4b7d0b.gatsbyjs.io/docs/browser/browser-monitoring/getting-started/introduction-browser-monitoring/embed/

**Review by sunnyzanchi — APPROVED** [review]

**Pull request merged**